Release Notes/510.md

```diff
@@ -3,11 +3,17 @@
 ## New APIs
 
 - `SyntaxStringInterpolation.appendInterpolation(_: (some SyntaxProtocol)?)`
-  - Descriptions: Allows optional syntax nodes to be used inside string interpolation of syntax nodes. If the node is `nil`, nothing will get added to the string interpolation.
+  - Description: Allows optional syntax nodes to be used inside string interpolation of syntax nodes. If the node is `nil`, nothing will get added to the string interpolation.
   - Pull Request: https://github.com/apple/swift-syntax/pull/2085
 - `SyntaxCollection.index(at:)`
   - Description: Returns the index of the n-th element in a `SyntaxCollection`. This computation is in O(n) and `SyntaxCollection` is not subscriptable by an integer.
   - Pull Request: https://github.com/apple/swift-syntax/pull/2014
+- `ClosureCaptureSyntax.init(leadingTrivia: Trivia? = nil, specifier: ClosureCaptureSpecifierSyntax? = nil, name: TokenSyntax, equal: TokenSyntax = TokenSyntax.equalToken(), expression: some ExprSyntaxProtocol, trailingComma: TokenSyntax? = nil, trailingTrivia: Trivia? = nil)`
+  - Description: Provides a convenience initializer for `ClosureCaptureSyntax` that takes a concrete `name` argument and automatically adds `equal = TokenSyntax.equalToken()` to it.
+  - Pull Request: https://github.com/apple/swift-syntax/pull/2127
+- `EnumCaseParameterSyntax.init(leadingTrivia: Trivia? = nil, modifiers: DeclModifierListSyntax = [], firstName: TokenSyntax, secondName: TokenSyntax? = nil, colon: TokenSyntax = TokenSyntax.colonToken(), type: some TypeSyntaxProtocol, defaultValue: InitializerClauseSyntax? = nil, trailingComma: TokenSyntax? = nil, trailingTrivia: Trivia? = nil)`
+  - Description: Provides a convenience initializer for `EnumCaseParameterSyntax` that takes a concrete `firstName` value and adds `colon = TokenSyntax.colonToken()` automatically to it.
+  - Pull Request: https://github.com/apple/swift-syntax/pull/2112
```

Comment:
The issue related to these changes contains important information about the new APIs. I would consider adding a link to it, as stated in the template below. What do you think about it? 
```suggestion
- `ClosureCaptureSyntax.init(leadingTrivia: Trivia? = nil, specifier: ClosureCaptureSpecifierSyntax? = nil, name: TokenSyntax, equal: TokenSyntax = TokenSyntax.equalToken(), expression: some ExprSyntaxProtocol, trailingComma: TokenSyntax? = nil, trailingTrivia: Trivia? = nil)`
  - Description: Provides a convenience initializer for `ClosureCaptureSyntax` that takes a concrete `name` argument and automatically adds `equal = TokenSyntax.equalToken()` to it.
  - Issue: https://github.com/apple/swift-syntax/issues/1984
  - Pull Request: https://github.com/apple/swift-syntax/pull/2127
- `EnumCaseParameterSyntax.init(leadingTrivia: Trivia? = nil, modifiers: DeclModifierListSyntax = [], firstName: TokenSyntax, secondName: TokenSyntax? = nil, colon: TokenSyntax = TokenSyntax.colonToken(), type: some TypeSyntaxProtocol, defaultValue: InitializerClauseSyntax? = nil, trailingComma: TokenSyntax? = nil, trailingTrivia: Trivia? = nil)`
  - Description: Provides a convenience initializer for `EnumCaseParameterSyntax` that takes a concrete `firstName` value and adds `colon = TokenSyntax.colonToken()` automatically to it.
  - Issue: https://github.com/apple/swift-syntax/issues/1984
  - Pull Request: https://github.com/apple/swift-syntax/pull/2112
```

Reply:
Yes, I think linking the issue would be valuable. 

Also, I think the full API signature isn’t that enlightening. What I would do instead is name the items
- `ClosureCaptureSyntax.init` with a required `name`
- `EnumCaseParameterSyntax.init` with a required `firstName`

Reply:
How about incorporating release notes into the DocC documentation for `SwiftSyntax` package? With this approach, it would be really easy to add direct links to specific methods right in the code.

Reply:
The problem is that we would need to decide which module to put the release notes in. If we put it in the SwiftSyntax module, you can’t link to any API in e.g. `SwiftParser`. Since the release notes are for the entire package and not one specific module, I think having them at a top-level makes sense for now.

Reply:
FWIW https://github.com/apple/swift-docc/issues/208 is tracking this restriction.

Reply:
Thank you both for the review! I'll clean up the changelog for now. Hopefully, I can get a generator-based version quickly enough and clean them up altogether in a few days.